Tests/SonarRunner.Shim.Tests/SonarRunnerWrapperTests.cs

```diff
@@ -72,12 +89,24 @@ public void SonarRunner_AdditionalArgumentsPassed()
             string propertiesFilePath = Path.Combine(TestUtils.CreateTestSpecificFolder(this.TestContext), "analysis.properties");
             File.CreateText(propertiesFilePath);
 
+            string[] userArgs = new string[] { "/Dsonar.login=me", "/Dsonar.password=my.pwd" };
+
             // Act
-            bool success = SonarRunnerWrapper.ExecuteJavaRunner(logger, exePath, propertiesFilePath);
+            bool success = SonarRunnerWrapper.ExecuteJavaRunner(logger, exePath, propertiesFilePath, userArgs);
             Assert.IsTrue(success, "Expecting execution to succeed");
-            logger.AssertSingleMessageExists("Command line args:", SonarRunnerWrapper.AdditionalRunnerArguments);
+            string message = logger.AssertSingleMessageExists("Command line args:", "/Dsonar.login=me", "/Dsonar.password=my.pwd");
```

Comment:
The Java sonar-runner is being called via the .Net ProcessRunner, which doesn't log secrets. The Java sonar-runner itself may log the arguments it is passed - we'll need to check.
However, we aren't executing the Java sonar-runner here; the secrets are being output to the console by the batch file.